pkg/init/init

```diff
@@ -2,32 +2,11 @@
 
 setup_console() {
 	tty=${1%,*}
-	speed=${1#*,}
 	inittab="$2"
-	securetty="$3"
-	line=
-	term="linux"
-	[ "$speed" = "$1" ] && speed=115200
-
-	case "$tty" in
-	ttyS*|ttyAMA*|ttyUSB*|ttyMFD*)
-		line="-L"
-		term="vt100"
-		;;
-	tty?)
-		line=""
-		speed="38400"
-		term=""
-		;;
-	esac
 	# skip consoles already in inittab
 	grep -q "^$tty:" "$inittab" && return
 
 	echo "$tty::once:cat /etc/issue" >> "$inittab"
```

Comment:
I like the idea of the whale there; real branding. 

Without it in init, it makes it optional for people. I think the branding is important and should be part of init. Make people work to _remove_ it (which they can be using their own init or mapping a different `/etc/issue`) rather than _add_ it.

> It makes it much easier to rework the base init stuff if there is less here.

I agree with that.